docs/Building-and-Running-Firmware.md

```diff
@@ -23,7 +23,7 @@ There are two recomended options for cloning the `AMDC-Firmware` repo from GitHu
 
 ### Open-Source Example Applications
 
-If you are _not_ creating private user applications, i.e. your code will be contributed back to the `AMDC-Firmware` repo as an example application:
+If you are _not_ creating private user applications, i.e. your code will _not_ be contributed back to the `AMDC-Firmware` repo as an example application:
```

Comment:
This is a confusing sentence, double negative. I think it was right initially.

It is trying to say: "If you are making example open-source code (so it will go back to the main repo), do the following:"

Reply:
Ha-- disregard this PR, I was confused.